src/indexCors.ts

```diff
@@ -78,7 +79,8 @@ app.all('*', async (req: Request, res: Response) => {
       if (name === 'content-encoding') return
       res.header(name, value)
     })
-    res.status(response.status).send(bodyText)
+    const overrideStatusCode = checkStatusCode(response)
+    res.status(overrideStatusCode ?? response.status).send(bodyText)
```

Comment:
I'd send a uniform response for 418 requests. Including a body like `Rate limited` always makes it a simple uniform response to the client and obvious to us during debuggin.